pkg/cmd/install/hubaddon/scenario/resources.go

```diff
@@ -7,3 +7,56 @@ import (
 
 //go:embed addon
 var Files embed.FS
+
+var (
+	PolicyFrameworkConfigFiles = []string{
+		"addon/policy/addon-controller_clusterrole.yaml",
+		"addon/policy/addon-controller_clusterrolebinding.yaml",
+		"addon/policy/addon-controller_role.yaml",
+		"addon/policy/addon-controller_rolebinding.yaml",
+		"addon/policy/addon-controller_serviceaccount.yaml",
+		"addon/policy/policy.open-cluster-management.io_placementbindings.yaml",
+		"addon/policy/policy.open-cluster-management.io_policies.yaml",
+		"addon/policy/policy.open-cluster-management.io_policyautomations.yaml",
+		"addon/policy/policy.open-cluster-management.io_policysets.yaml",
+		"addon/policy/propagator_clusterrole.yaml",
+		"addon/policy/propagator_clusterrolebinding.yaml",
+		"addon/policy/propagator_role.yaml",
+		"addon/policy/propagator_rolebinding.yaml",
+		"addon/policy/propagator_service.yaml",
+		"addon/policy/propagator_serviceaccount.yaml",
+		"addon/policy/clustermanagementaddon_configpolicy.yaml",
+		"addon/policy/clustermanagementaddon_policyframework.yaml",
+		"addon/appmgr/crd_placementrule.yaml",
```

Comment:
This PlacementRule CRD is a special case that needs to be handled and should only be cleaned up if _both_ addons are uninstalled.

Reply:
does policy still have a hard deps on placementRule, or we just do not create this crd when installing?

Reply:
Currently the governance-policy-propagator controller won't start properly unless the PlacementRule CRD exists.

Reply:
FYI: In the next OCM release, Policy no longer has a dependency on PlacementRule, so addon CRD cleanup can proceed in `clusteradm`.